lib/iex/lib/iex/info.ex

```diff
@@ -43,6 +43,18 @@ defimpl IEx.Info, for: Atom do
     [{"Data type", "Atom"}] ++ description ++ specific_info
   end
 
+  defp module?(atom) do
+    case :code.get_object_code(atom) do
+      :error ->
+        Code.ensure_loaded?(atom)
+
+      {^atom, beam, _path} ->
+        info = :beam_lib.info(beam)
+
+        {:ok, atom} == Keyword.fetch(info, :module)
```

Comment:
Personally, I feel more comfortable with this check being the other way around for some reason. If you also think it's a good change, here's a nice little change that you can just commit otherwise no problem! 😄 

```suggestion
        Keyword.fetch(info, :module) == {:ok, atom}
```